src/app/auth/create-user/page.tsx

```diff
@@ -0,0 +1,19 @@
+import Container from '@/components/layout/Container';
+
+import CreateUser from '../components/CreateUserForm';
+
+export default function Create() {
+  return (
+    <>
+      <div className='space-y-4 mt-8'>
+        <h2 className='font-arvo text-3xl text-center'>Join Monkeys</h2>
```

Comment:
There is already a PageHeader component present, we can use that here for add heading and subheading.

Reply:
done